sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -33,9 +34,11 @@ import { ECertFullTimeResponseRecord } from "./e-cert-full-time-integration/e-ce
 import { ProcessSFTPResponseResult } from "../models/esdc-integration.model";
 import { ESDCIntegrationConfig } from "../../types";
 import { ESDCFileResponseDTO } from "../../route-controllers/esdc-integration/models/esdc-model";
+import { DisbursementScheduleWithStopFullTimeBCFundingStatus } from "../../services/disbursement-schedule-service/disbursement-schedule.models";
```

Comment:
DisbursementScheduleWithStopFullTimeBCFundingStatus Please suggest a small name :)

Reply:
Is this specific for the ECert generation, right?
It can be something like `DisbursementScheduleECert` or `ECertDisbursementSchedule`.